c_src/ffmpeg_video_compositor/filter.c

```diff
@@ -0,0 +1,99 @@
+#include "filter.h"
```

Comment:
Please provide documentation for functions, especially for those used in other files.

Reply:
Done